Formula/pip-audit.rb

```diff
@@ -0,0 +1,129 @@
+class PipAudit < Formula
+  include Language::Python::Virtualenv
+
+  desc "Audits Python environments and dependency trees for known vulnerabilities"
+  homepage "https://pypi.org/project/pip-audit/"
+  url "https://github.com/trailofbits/pip-audit/archive/refs/tags/v1.0.1.tar.gz"
+  sha256 "15ff2445ed2edb22dc7665366ad66563f5deee8dba0cc92e57b7ba9a421c97c6"
```

Comment:
```suggestion
  url "https://files.pythonhosted.org/packages/64/39/07bba0cff03c8126ab1ac319f00746ccc24c784ff8b4ae14963d440d4173/pip-audit-1.0.1.tar.gz"
  sha256 "351f7f279ffb20cfad3fe876e84bf524a87aaa2a8830a4cad66f24f30b2a68f6"
```
This works better with our tooling for updating resources automatically.